src/boot/efi/boot.c

```diff
@@ -383,6 +384,10 @@ static VOID print_status(Config *config, CHAR16 *loaded_image_path) {
                 FreePool(b);
         }
 
+        if (shim_loaded()) {
+                Print(L"Shim:                   present\n");
+        }
```

Comment:
Please follow coding style, i.e. no {} around single-line if blocks. For details see CODING_STYLE